visualization/app/codeCharta/util/fileHelper.ts

```diff
@@ -40,6 +41,14 @@ function getAttributeTypes(attributeTypes: AttributeTypes | OldAttributeTypes):
 	}
 }
 
+function getAttributeDescriptors(attributeDescriptors: AttributeDescriptors): AttributeDescriptors {
+	if (!attributeDescriptors) {
+		return {}
+	}
+
+	return attributeDescriptors
```

Comment:
I think this will work also
```suggestion
	return attributeDescriptors || {}
```
Or using the ternary operator would be fine for me as well.